pkg/asset/internal/templates.go

```diff
@@ -1040,7 +1040,7 @@ var EtcdTPRTemplate = []byte(`{
       "static": {
         "member": {
           "peerSecret": "etcd-member-peer-tls",
-          "clientSecret": "etcd-member-client-tls"
+          "serverSecret": "etcd-member-client-tls"
```

Comment:
nit: in the etcd-operator docs the secret name is slightly different (don't have "member"). https://github.com/coreos/etcd-operator/blob/master/doc/user/cluster_tls.md#static-cluster-tls-policy

but that can be a follow up too